bench-tps/tests/bench_tps.rs

```diff
@@ -29,8 +33,22 @@ use {
     std::{sync::Arc, time::Duration},
 };
 
+fn program_account(elf: &[u8]) -> AccountSharedData {
```

Comment:
Can you document a little on what is the elf? or maybe rename to elf_code?

Reply:
how about `program_data`?